b2share/modules/deposit/errors.py

```diff
@@ -33,3 +35,59 @@ class B2ShareDepositError(RESTValidationError):
 
 class InvalidDepositError(B2ShareDepositError):
     """Exception raised when the Deposit is invalid."""
+
+
+class VersioningDepositError(B2ShareDepositError):
+    """Versioning related exception."""
+
+
+class OwnershipVersioningError(VersioningDepositError):
+    """Exception raised when ownership changed in a version chain.
+    Currently the ownership must be a constant."""
+    code = 403
```

Comment:
We need to be sure that this is not raised if a user is anonymous (not authenticated), as it would then be 401 and not 403.